compiler/circle-mpqsolver/README.md

```diff
@@ -0,0 +1,49 @@
+# circle-mpqsolver
+_circle-mpqsolver_ provides light-weight methods for finding a high-quality mixed-precision model within a reasonable time.
+
+## bisection
+So assuming the model is parameterized by single parameter (let's call it _depth_), we can split the model so that all layers with _depth_ parameter below _specified_depth_ will be in the _input_group_, while those layers with _depth_ above _specified_depth_ will be in the _output_group_ (node's _depth_ mimics distance from _input_ to the node).
```

Comment:
please split lines to come in 80 or 100 columns for better editing.

Reply:
ok.